lib/shared/permissions.js

```diff
@@ -143,14 +158,18 @@ exports.getEnvironmentCommandPrefix = (environment) => {
  * });
  */
 exports.elevateCommand = (command, options) => {
-  const prefixedCommand = _.concat(exports.getEnvironmentCommandPrefix(options.environment), command);
+  const prefixedCommand = _.concat(
+    exports.getEnvironmentCommandPrefix(options.environment),
+    _.map(command, quoteString)
```

Comment:
You were right. After some experimentation, the quoting here is only needed for Windows, given that `command-join` takes care of it for OS X later on.